packages/core/src/Site/index.js

```diff
@@ -436,14 +436,14 @@ class Site {
       .filter(addressablePage => !addressablePage.src.startsWith('_'))
       .forEach((page) => {
         const addressablePagePath = path.join(this.rootPath, page.src);
-        const relativePagePathWithoutExt = fsUtil.removeExtension(
+        const relativePagePathWithoutExt = fsUtil.removeExtensionPosix(
```

Comment:
Try to rebase to master so this PR doesn't overwrite the changes that has already been merged.